pkg/api/types.go

```diff
@@ -1669,6 +1671,14 @@ func (a *AgentPoolProfile) GetKubernetesLabels(rg string, deprecated bool) strin
 	return buf.String()
 }
 
+// IsCSIProxyEnabled returns true if csi proxy service should be enable for Windows nodes
+func (w *WindowsProfile) IsCSIProxyEnabled() bool {
+	if w.EnableCSIProxy != nil {
+		return *w.EnableCSIProxy
+	}
+	return DefaultEnableCSIProxyWindows
```

Comment:
this needs to be `common.DefaultEnableCSIProxyWindows` (assuming `common` is a reference to the api/common package)